locales/en-US/server.ftl

```diff
@@ -123,9 +126,17 @@ shotPageDownloadShot =
     .title = Download
 shotPageEditButton =
     .title = Edit this image
-shotPagefavoriteButton =
+shotPageFavoriteButton =
     .title = Favorite this shot
+shotPageBackToHomeButton =
+    .title = Homepage
+shotPageAllShotsButton =
+    .title = All Shots
+shotPageAllShots = All Shots
 shotPageDownload = Download
+shotPageDraw = Draw
+shotPageFavorite = Favorite
```

Comment:
Same question: verb or noun?

Reply:
It's a verb, added comment to highlight that